src/smart-components/workflow/edit-workflow-info-modal.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { withRouter } from 'react-router-dom';
+import { ActionGroup, Button, FormGroup, Modal, Split, SplitItem, Stack, StackItem } from '@patternfly/react-core';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications';
+import { addWorkflow, updateWorkflow, fetchWorkflow } from '../../redux/actions/workflow-actions';
+import { WorkflowStageLoader } from '../../presentational-components/shared/loader-placeholders';
+import StageInformation from './add-stages/stage-information';
+import '../../App.scss';
+
+const EditWorkflowInfoModal = ({
+  history: { push },
+  match: { params: { id }},
+  addNotification,
+  fetchWorkflow,
+  updateWorkflow,
+  postMethod,
+  workflow,
+  isFetching
+}) => {
+  const [ formData, setValues ] = useState({});
+
+  const handleChange = data => {
+    setValues({ ...formData, ...data });
+  };
+
+  useEffect(() => {
+    fetchWorkflow(id).then((data) => { setValues({ ...formData, ...data.value });});
+  }, []);
+
+  const onSave = () => {
+    const { name, description } = formData;
+    const workflowData = { name, description };
+    updateWorkflow({ id, ...workflowData }).then(postMethod()).then(push('/workflows'));
```

Comment:
You are creating workFlowData one line above and then you are destructuring it here. What about to do:

```jsx
const workflowData = { id, name, description };
updateWorkflow(workflowData)......
```